content/docs/concepts/jobs_instances.md

```diff
@@ -5,9 +5,8 @@ sort_rank: 3
 
 # Jobs and instances
 
-In Prometheus terms, any individually scraped target is called an _instance_,
-usually corresponding to a single process. A collection of instances of the
-same type (replicated for scalability or reliability) is called a _job_.
+In Prometheus terms, a destination you can scrape is called an _instance_,
```

Comment:
I don't think there should be any mention of targets on this page. I'd focus on instances being inside a job.